tests/test_security_path_traversal.py

```
@@ -204,3 +204,35 @@ def test_run_tailer_path_validation(tmp_path):
+    # Backslash handling on Windows vs Linux is tricky, but normalized to / in function for check
+    # We test with forward slashes which work everywhere in python
+    assert validate_relative_path("foo/bar/baz") == "foo/bar/baz"
+
+    # Invalid absolute paths
+    with pytest.raises(ValueError, match="must be relative"):
+        # We need an absolute path. On linux /etc/passwd is absolute.
+        # On windows C:\Windows is absolute.
+        # We can construct one using pathlib to be safe.
+        import pathlib
+        abs_path = str(pathlib.Path.cwd() / "foo")
+        validate_relative_path(abs_path)
```

Ruff (0.15.2)

[warning] `pytest.raises()` block should contain a single simple statement

(PT012)